src/patterns/components/offline-notification/overview.hbs

```diff
@@ -0,0 +1,7 @@
+---
+order: 10
+notes: |
+  In page notification about lack of internet connectivity.
+---
+
+{{{embed "patterns.components.offline-notification.example"}}}
```

Comment:
Pagination is set up in that way because we wanted to show it with a "current page" of `2` so both arrows were visible, but it would be kind of a pain to include a template that always defaulted to `2` instead of `1` as the default page.

Any pattern is embeddable, so `{{{embed "patterns.components.offline-notification.overview"}}}` would work just fine.